### DISCUSSION TITLE
Typescript : Error message when using formState :  {errors }

### DISCUSSION THREAD

**Discussion by dhieunguyen** [react-hook-form/react-hook-form#5789] [Q&A] [answered]
I'm using v7.9.0 and my code warning me like this

![72d8fd42-ba54-494f-a560-a3d7e6178a53](https://user-images.githubusercontent.com/64343185/123902588-4d979180-d997-11eb-925c-b46c39006c22.png)

**Comment by bluebill1049** [comment, marked as answer]
can you provide a simple codesandbox? it's probably to do with your type.

**Reply by dhieunguyen** [reply]
i'm very sorry :( my React Native code has some components. Can i give u screenshots ?

**Reply by bluebill1049** [reply]
TS or JS? and can you hover over for the error message.

**Reply by dhieunguyen** [reply]
i use TS

**Reply by dhieunguyen** [reply]
![image](https://user-images.githubusercontent.com/64343185/123903951-d44d6e00-d999-11eb-9d3c-88f283a2606f.png)
this is my rules

**Reply by dhieunguyen** [reply]
and my formvalue 
![image](https://user-images.githubusercontent.com/64343185/123903997-e8916b00-d999-11eb-8c77-c44358f33d85.png)

**Reply by bluebill1049** [reply]
I think you are missing the generic at `useForm<...>`

**Reply by dhieunguyen** [reply]
yes. But idk to use any generic for useForm

**Reply by bluebill1049** [reply]
`useForm<FormValue>()` ?

**Reply by dhieunguyen** [reply]
i got it :( my bad. Thanks for your support 💯

**Reply by bluebill1049** [reply]
np
